server/interceptor/interceptor.go

```diff
@@ -248,7 +248,18 @@ type rpcInfo struct {
 // TreeType and Readonly are determined based on the request type.
 func getRPCInfo(req interface{}, quotaUser string) (*rpcInfo, error) {
 	switch req.(type) {
-	case *quotapb.CreateConfigRequest,
+	case
+		// Do not intercept certain admin requests:
+		// * CreateTree creates new trees (no associated treeID)
+		// * Get, Delete and UndeleteTree may read with deleted trees
+		// * ListTrees returns 0 or more trees
```

Comment:
I don't really follow why this needed to change (although TBH the whole interceptor thing has become too much for me to follow...)

Reply:
It seemed simpler to do it this way.

I've been wanting to have a second go at it and try to simplify things a bit. No promises, though.